microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -276,8 +275,9 @@ def kill_previous_testpmd_command(self) -> None:
             "Timer process was not initialized before "
             "calling kill_previous_testpmd_command"
         ).is_not_none()
-        self.node.execute(f"killall -s INT {self.command}", sudo=True, shell=True)
-        self.timer_proc.kill()
+        self.node.execute(
+            f"killall -s INT {self.command}", sudo=True, shell=True, timeout=40
```

Comment:
Because the previous processes are not useful. It can be killed by 9, which responds fast.

Reply:
Now that you mention it trying both would make sense.

Reply:
I totally missed that kill optionally allows other signals with timeout
how does `kill -s INT testpmd --timeout 5000 TERM --timeout 10000 KILL` sound?

Reply:
Ah nevermind, not supported on all versions of kill

Reply:
Kill by 9 doesn't wait for any response, so it's fast. Send message to process let it finalize. Because this place is to clean up before start, so it doesn't need to wait. But it's better to kill after each test run as well. The test environment may be used for next test case, the next test case may not be dpdk, so the leaving process affects next one.

Reply:
I had it in my head that INT was needed but it seems like KILL actually is fine as long as we don't care about the exit code of testpmd. I'll swap to KILL